lib/translator.js

```diff
@@ -36,19 +38,37 @@ Translator.prototype.getLocales = function() {
   return this._locales;
 };
 
+var createPoParseFn = function(lang) {
+  return function(msgDir) {
+    var fp = path.join(msgDir, lang, 'LC_MESSAGES', 'messages.po');
+    var po = '';
+    try{
+      po = fs.readFileSync(fp, {encoding: 'utf8'});
+    }catch(err) {
+      if(err.code !== 'ENOENT') {
```

Comment:
setting `po` to empty string results in this function returning `undefined`.  Lets make our functions always return useable results.

Suggesting:
```js
var hasPo = fs.existsSync(fp);
var po = hasPo ? fs.readFileSync(fp, {encoding: 'utf8'}) : null;
var catalog = po ? gettextParser.po.parse(po).translations[''] : {};
return catalog;
```